src/cmd/report.go

```diff
@@ -142,24 +143,24 @@ func fetchGitHubData(repo string) {
 
 }
 
-func fetchDockerHubData(image string) {
+func reportDockerHubData(image string) {
 
 	pulls, err := docker.ImagePulls(image)
 	if err != nil {
-		log.Fatal("Error: Failed to fetch Docker Hub pulls.")
+		log.Fatal("Error: Failed to report Docker Hub pulls.")
 	}
 
 	stars, err := docker.ImageStars(image)
 	if err != nil {
-		log.Fatal("Error: Failed to fetch Docker Hub stars.")
+		log.Fatal("Error: Failed to report Docker Hub stars.")
```

Comment:
These functions should still be fetch as they is what they are doing. The command is reporting, not these functions.